about.html

```diff
@@ -0,0 +1,121 @@
+			<ul class="nav-menu">
+				<li class="about"><a href="about.html">About</a></li>
+				<li><a href="index.html">Home</a></li>
+				<li><a href="#">Program</a></li>
+				<li><a href="#">Join</a></li>
+				<li><a href="#">Sponsors</a></li>
+				<li><a href="#">News</a></li>
+				<li id="camp"><a href="#">Campaign</a></li>
+			</ul>
+		</div>
+	</section>
+	<div class="desktop-p">
+		<p class="moto">Keeping You Healthy</p>
+		<h1> RAY'S ELEMENT AQUA <br /> GLOBAL SUMMIT 2021 </h1>
+	</div>
+	<p class="desktop-article">Element Aqua Summit is the most highly influential and profitable network for
+		Carribean's health,fitness and wellness industry.
+		we organize health, fitness and wellness conferences, events that moves even the most fable minds to
+		achieve their highest potential as far as living healthy is concerned. Our Awards duely recognise
+		leading companies and excutives from accross the health, fitness and wellness ecosystem.
+	</p>
+	<div class="article2">
+		<p>Please reach us through email for any futher questions and informations you need</p>
+		<a class="reach-out" href="mailto">rayselementsummit2021@aqua.org</a>
+	</div>
+	<section class="logo-wrapper">
+		<div class="aqua-logo">
+			<p id="logo-p">The Ray's Element Aqua Summit 2021 logo</p>
+		</div>
+		<div class="text">
+			<p class="mobile-text"> The logo of Ray's Aqua summit 2021 was decided through a series of logo competition
+				by stake holders from 8 may to 12 june.</p>
+		</div>
+		<p id="text-logo"> The logo of Ray's Aqua summit 2021 was decided through a series of logo competition
+			by stake holders from 8 may to 12 june. After online public voting and selection commitee's judging
+			process, the logo from photographer Vincent won the price. After consultation with Mr john we have
+			optimized the logo for this year's summit as follow</p>
+		<div class="mobile-image">
+			<img class="logo-pic" src="Assets/Icons/running.jpeg" alt="summit-logo">
+		</div>
+	</section>
+	<hr id="line">
+	<div class="past-summits">
+		<h3>See the past RAY'S Aqua Global Summits</h3>
+		<p>Take a look at the last two summits which took place in Dominincan repulblic and Barbados</p>
+	</div>
+	<section class="Summit-examples">
+		<div class="images">
+			<img src="Assets/Images/barbardos.jpeg" alt="barbados-summit">
+			<img id="d-down" src="Assets/Images/dominican.jpeg" alt="carribean-summit">
+		</div>
+	</section>
+	<section class="partner">
+		<div class="footer-mobile">
+			<h2>Partners</h2>
+			<ul class="partners-mobile">
+				<li><a href="https://www.mozilla.org/en-US/firefox/new/" id="dark">mozilla</a></li>
+				<li><a href="https://www.google.com/">Google</a></li>
+				<li><a href="https://www.naver.com/" id="lightgrey">NAVER</a></li>
+				<li><a href="https://www.crunchbase.com/organization/daum-kakao">daumkakao</a></li>
+				<li><a href="https://www.airbnb.com/">Airbnb</a></li>
+			</ul>
+		</div>
+	</section>
+	<footer class="footer-container">
```

Comment:
- [ ] Kindly use the same background colour used in the template design. It should be `#fafafa` not `#f2f7f6`
- [ ] Also, reduce the font weight of the section header.
_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/125092018-c082c580-e085-11eb-99bb-fba92948878a.png) | ![image](https://user-images.githubusercontent.com/49621279/125092335-19eaf480-e086-11eb-9cec-10200b3c8fcb.png)

Reply:
the weight of this section header was reduced until one of you told me to increase it, all you need to do is to follow the thread here to see it.  and this color you are asking me to use isn't provided in any part of the project design or template. they are three colors in Behance and I will list them here
#272131 which is light black
#ec5242 which is light red
#d3d3d3 which is kinda white

the background color that I have chosen is one I feel was more consistent with the design as none of the three main provided colors would work well with that section.  

here comes what I think everyone can improve upon, look at the last changes requested by the last CR before asking for more changes.

Reply:
- [ ] The color used as the background in this section does not feature in any of the 3 
main colours used in the design template.

- [ ] The issue of font weight, I've added an image of the actual design here for reference.

Reply:
> * [x]  The color used as the background in this section does not feature in any of the 3
>   main colours used in the design template.
> * [x]  The issue of font weight, I've added an image of the actual design here for reference.

Neither does it feature in the template original design. regardless I still went ahead and use the background color as I want this approved and can't go into any more details debating this. this all review has cost me my sleep and productivity, demotivated me, and shit. 

the font-weight if you spend five minutes from the first review, you will see where he asked me to increase it as it was too light or small. you need to remember also the font-weight of text will differ on any machine you check it from, this comes from the resolution of each screen.